site/content/posts/announcing-kudo-0.3.0.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Announcing KUDO 0.3.0
+date: 2019-06-28
+---
+
+We are proud to announce the release of KUDO v0.3.0! This release focuses on core infrastructure inside of KUDO as the format for developing operators and running operators stabilizes.
```

Comment:
This is the same sentence we used in the v0.2.0 announcement. For a second I felt lost. Let's give a one-sentence summary of what's new instead.